code/game/machinery/shieldgen.dm

```diff
@@ -511,22 +514,18 @@
 
 //////////////Containment Field START
 /obj/machinery/shieldwall

```

Comment:
That's why I wrote it should be machinery/power in my last PR about this crappy thing